cmd/launcher/launcher.go

```diff
@@ -283,6 +285,17 @@ func runLauncher(ctx context.Context, cancel func(), multiSlogger, systemMultiSl
 	go checkpointer.Once(ctx)
 	runGroup.Add("logcheckpoint", checkpointer.Run, checkpointer.Interrupt)
 
+	watchdogController, err := watchdog.NewController(ctx, k)
+	if err != nil { // log any issues here but move on, watchdog is not critical path
+		slogger.Log(ctx, slog.LevelError,
+			"could not init watchdog controller",
+			"err", err,
+		)
+	} else if watchdogController != nil { // watchdogController will be nil on non-windows platforms for now
+		k.RegisterChangeObserver(watchdogController, keys.LauncherWatchdogEnabled)
+		runGroup.Add("watchdog_controller", watchdogController.Run, watchdogController.Interrupt)
```

Comment:
Nitpick -- all the other rungroups are camel case (probably not a great choice, my b lol). Can we make them consistent in one direction or another?

Reply:
oh yep! I will update this to be camel case